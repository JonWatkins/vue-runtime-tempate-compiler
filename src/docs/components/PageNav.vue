<template>
  <div class="page-nav">
    <div class="page-nav--inner clearfix">
      <div class="float-left">
        <slot name="left" />
      </div>
      <div class="float-right">
        <slot name="right" />
      </div>
    </div>
  </div>
</template>

<script>
export default {
  name: 'PageNav'
}
</script>

<style lang="scss" rel="stylesheet/scss">
.page-nav {
  width: 100%;
  padding: 2rem 0;

  &--inner {
    min-height: 1rem;
    margin-top: 0;
    border-top: 1px solid #eaecef;
    padding-top: 1rem;
    overflow: auto;
  }
}
</style>
