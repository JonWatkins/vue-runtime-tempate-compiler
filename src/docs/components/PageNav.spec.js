import { shallowMount } from '@vue/test-utils'
import PageNav from './PageNav.vue'
const { localVueInst } = global

let wrapper

describe('PageNav', () => {
  beforeEach(() => {
    wrapper = shallowMount(PageNav, {
      localVue: localVueInst
    })
  })

  it('is called', () => {
    expect(wrapper.name()).toBe('PageNav')
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  it('render correctly', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
