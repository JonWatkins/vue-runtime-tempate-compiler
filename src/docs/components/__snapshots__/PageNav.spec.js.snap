// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNav render correctly 1`] = `
<div class="page-nav">
  <div class="page-nav--inner clearfix">
    <div class="float-left"></div>
    <div class="float-right"></div>
  </div>
</div>
`;
